test/inproc.js

```diff
@@ -15,6 +15,8 @@ test('inproc socket pub sub', function (t) {
     var addr = 'inproc://pubsub';
     var msg = 'hello world';
 
+    sub.chan('');
```

Comment:
> that would be useful. maybe if we run a setTimeout or Interval after sub socket is created that checks for the existence of any length on this.channel.

Then you run into the polling problem.  What duration do you use? If too long then you miss messages, if too short then you are subject to v8/machine perf.  Too brittle.

So, if you don't subscribe to any channels explicitly, sub sockets don't work (can we confirm)?

> once we've subscribed to '', there's no point in registering subsequent subscription topics

Can we make `''` the default, then the first time someone calls `chan`, can we `rmchan('')`?